project/versioning.scala

```diff
@@ -0,0 +1,37 @@
+object Versioning {
```

Comment:
Please leave a doc comment describing the purpose of this file.

Reply:
done